public/js/all.js

```diff
@@ -376,3 +376,19 @@ function initializeTextareaCounter() {
 }
 
 window.addEventListener('load', initializeTextareaCounter);
+
+// FIXME: Rename me.
+/**
+ * @param {Event} event
+ */
+function foo(event) {
+  const minimumScrollPosition = event.target.offsetHeight;
+  const scrollPosition = event.target.scrollTop + event.target.offsetHeight;
+
+  const topFadeOpacity = Math.min((scrollPosition - minimumScrollPosition) / 180, 1.0);
+  const bottomFadeOpacity = Math.min((event.target.scrollHeight - scrollPosition) / 180, 1.0);
+
+  event.target.querySelector('.awards-fade-top').style.opacity = topFadeOpacity;
+  event.target.querySelector('.awards-fade-bottom').style.opacity = bottomFadeOpacity;
```

Comment:
We need to cache these rather than querying them on every frame. This could be problematic if there are 2 awards sections that both need to be scrolled. We probably need a different query, maybe ID > child class